pyiron_gui/project/project_browser.py

```diff
@@ -140,40 +495,34 @@ def __getattr__(self, item):
     def clear_output(self, *args, **kwargs):
         clear_button = kwargs.pop('clear_button', False)
         if clear_button:
-            self.buttons = widgets.HBox()
+            self.header = widgets.HBox()
         self.output.clear_output(*args, **kwargs)
         self.refresh()
 
-    def _update_buttons(self, obj):
-
-        def click_button(b):
-            self.output.clear_output()
-            self.display(obj=obj)
-
-        if isinstance(obj, PyironWrapper) and obj.has_self_representation:
-            button = widgets.Button(description="Re-plot " + obj.name)
-            button.on_click(click_button)
-            self.buttons.children = tuple([button])
-
-        self.refresh()
-
     def display(self, obj, default_output=None):
-        self._update_buttons(obj)
+        if isinstance(obj, BaseWrapper):
+            self.display(obj.gui, default_output=default_output)
+        elif isinstance(obj, np.ndarray):
+            self.display(NumpyWidget(obj), default_output=default_output)
```

Comment:
Is it overkill to make a `NumpyWrapper` as well?  I guess it wouldn't need the project back reference, etc., so it's ok if it is.

Reply:
Writing a `NumpyWrapper` is not a big thing. However, right now I pass the outcome of `project[item from project.list_nodes]` to this `display` function directly. This outcome might, thus, be of any type and for `np.ndarray` I happen to have a better representation. Therefore, I would need to shift a similar if statement into `_on_click_file` of the `ProjectBrowser` ([here](https://github.com/pyiron/pyiron_gui/blob/7d8e09759b3a5f76122bf192eb09fc71a3bbbe2e/pyiron_gui/project/project_browser.py#L894))